examples/09-Incremental-Training-with-Different-Learning-Rates-and-Layer-Freezing.ipynb

```diff
@@ -0,0 +1,853 @@
+{
```

Comment:
<p>The headings are not consistent with the previous example.</p><p> </p><p>There the main heading was Generate Synthetic Data</p><p> </p><p>Here you have Building a Two Tower Model ... -&gt; Generate Synthetic Data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/667/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>